pkg/acceptance/bettertestspoc/README.md

```diff
@@ -97,18 +97,54 @@ Resource config model builders can be generated automatically. For object `abc`
 - add object you want to generate to `allResourceSchemaDefs` slice in the `assert/resourceassert/gen/resource_schema_def.go`
 - to add custom (not generated) config builder methods create file `warehouse_model_ext` in the `config/model` package. Example would be:
 ```go
-func BasicWarehouseModel(
+func BasicAbcModel(
 	name string,
 	comment string,
-) *WarehouseModel {
-	return WarehouseWithDefaultMeta(name).WithComment(comment)
+) *AbcModel {
+	return AbcWithDefaultMeta(name).WithComment(comment)
 }
 
-func (w *WarehouseModel) WithWarehouseSizeEnum(warehouseSize sdk.WarehouseSize) *WarehouseModel {
+func (w *AbcModel) WithWarehouseSizeEnum(warehouseSize sdk.WarehouseSize) *AbcModel {
 	return w.WithWarehouseSize(string(warehouseSize))
 }
 ```
 
+### Adding new datasource config model builders
+Data source config model builders can be generated automatically. For object `abc` do the following:
+- add object you want to generate to `allDatasourcesSchemaDefs` slice in the `config/datasourcemodel/gen/datasource_schema_def.go`
+- to add custom (not generated) config builder methods create file `abc_model_ext` in the `config/datasourcemodel` package. Example would be:
+```go
+func BasicAbcModel(
+	name string,
+	comment string,
+) *AbcModel {
+	return AbcWithDefaultMeta(name).WithComment(comment)
+}
+
+func (w *AbcModel) WithWarehouseSizeEnum(warehouseSize sdk.WarehouseSize) *AbcModel {
```

Comment:
I think we could use a more standard data source method like `WithLike`, etc.

Reply:
It doesn't really matter for this example but changed.